pallets/loans/src/mock.rs

```diff
@@ -152,6 +152,9 @@ impl orml_tokens::Config for MockRuntime {
 parameter_types! {
 	pub const PoolPalletId: frame_support::PalletId = cfg_types::ids::POOLS_PALLET_ID;
 
+	/// The index with which this pallet is instantiated in this runtime.
+	pub PoolPalletIndex: u8 = <Pools as PalletInfoAccess>::index() as u8;
```

Comment:
What size is `Index` that we need to cast it? Will this be safe? Can we have test in case this is overflowing? ^^